kpi/template.adoc

```diff
@@ -1,20 +1,29 @@
-== X. KPI name
+== KPI-{counter:kpi}: my KPI
 
 === Status
 
 Draft|Approved
 
 === WCMP element(s)
 
-- list of XPaths
+* XPath 1
+* XPath 2
 
 === What is being measured
 
+Text description of measurement.
+
 === Rationale for measurement
 
+Why should this measurement take place.
+
 === Rules for implementation
 
 - use XPath and pseudo-code (if then else)
 - provide total possible score
 
+*Total possible score: x (100%)*
+
 === Guidance to score well on this assessment
```

Comment:
If we move xpaths, etc to this section. Change title to Guidance and Examples ....